deb/make_deb.sh

```diff
@@ -296,9 +299,9 @@ endif
 	mkdir $(installdir)/leo_gateway/cache
 	ln -s $(version) debian/tmp/usr/local/$(package)/current
 	cp leofs-adm $(bindir)
-	dpkg-shlibdeps $(installdir)/leo_manager_0/erts-*/bin/beam
 	mkdir -p debian/tmp/DEBIAN
 	dh_installdeb -Pdebian/tmp
+	dpkg-shlibdeps $(installdir)/leo_manager_0/erts-*/bin/{beam,erlexec} $$(find $(installdir) -name \*.so)
```

Comment:
As I commented on https://github.com/leo-project/leofs/issues/840#issuecomment-353497818, `find -executable` might be more perfect.